src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,170 @@
 package core.basesyntax;
 
+//import jdk.internal.util.ArraysSupport;
```

Comment:
Please, remove this comment.

Comments are useful when they explain very specific code. If your code is understandable by himself then remove comments.